platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/rest/DefaultHttpJsonRequest.java

```diff
@@ -181,48 +182,24 @@ protected DefaultHttpJsonResponse doRequest(int timeout,
                                                                                UnauthorizedException,
                                                                                ConflictException,
                                                                                BadRequestException {
-        final String authToken = getAuthenticationToken();
-        final boolean hasQueryParams = parameters != null && !parameters.isEmpty();
-        if (hasQueryParams || authToken != null) {
```

Comment:
Since authentication is dropped this change will change app behavior. Looks like it is not very safe for stability of the project